cluster-commands.go

```diff
@@ -393,3 +440,133 @@ func (adm *AdminClient) SRInternalGetIDPSettings(ctx context.Context) (info IDPS
 	err = json.Unmarshal(b, &info)
 	return info, err
 }
+
+// SRMetaInfo gets replication metadata
+type SRMetaInfo struct {
+	Enabled        bool
+	Name           string
+	DeploymentID   string
+	Buckets        map[string]SRBucketMetaInfo   // map of bucket metadata info
+	Policies       map[string]json.RawMessage    //  map of IAM policy name to content
+	UserPolicies   map[string]SRPolicyMapping    // map of username -> user policy mapping
+	GroupPolicies  map[string]SRPolicyMapping    // map of groupname -> group policy mapping
+	ReplicationCfg map[string]replication.Config // map of bucket -> replication config
+}
+
+// SiteReplicationMetaInfo - returns site meta info for replication.
+func (adm *AdminClient) SiteReplicationMetaInfo(ctx context.Context) (info SRMetaInfo, err error) {
```

Comment:
```suggestion
// ListSiteReplicationBuckets - returns site meta info for replication.
func (adm *AdminClient) ListSiteReplicationBuckets(ctx context.Context) (info SRBucketsResult, err error) {
```

Reply:
Then it should be perhaps `SRInfo`  not `SRMetaInfo` - and also `ListAllSites()`